jenkins/ci.suse.de/cloud-gating.yaml

```diff
@@ -92,8 +92,8 @@
           ses_enabled: true
           update_after_deploy: false
           tempest_filter_list: "\
-            keystone,swift,glance,cinder,neutron,nova,barbican,fwaas,lbaas,\
-            heat,magnum,manila"
+            smoke,keystone,swift,glance,cinder,neutron,nova,ceilometer,barbican,\
```

Comment:
we don't run `smoke` in the gate, mainly because these tests are already included in the other tempest filters

Reply:
Fixed